js/load.js

```diff
@@ -45,3 +46,6 @@ window.pkp = Object.assign(PkpLoad, {
 
 // Required by the URN plugin
 window.pkp.Vue.component('field-text', FieldText);
+
+// DOI
+window.pkp.Vue.component('DoiListPanel', DoiListPanel);
```

Comment:
We'll revisit this when we convert the panel to a page. That can be after an initial merge. We'll see how we get on.